lib/convert.js

```diff
@@ -274,7 +273,10 @@ module.exports = {
       }
       else {
         // Unhandled exception, rethrow
```

Comment:
when you change code, comments need to be changed too

Reply:
Also, as discussed, this bit of code should not change. We should ignore all ignore-able errors at the point where the exception is being thrown